compiler-plugin-tests/src/test/java/io/ballerina/stdlib/http/compiler/CompilerPluginTest.java

```diff
@@ -118,7 +84,7 @@ public void testInvalidMethodTypes() {
         diagnosticResult.diagnostics().forEach(result -> {
             if (result.diagnosticInfo().severity().equals(DiagnosticSeverity.ERROR)) {
                 Assert.assertEquals(result.diagnosticInfo().messageFormat(), REMOTE_METHODS_NOT_ALLOWED);
-                Assert.assertEquals(result.diagnosticInfo().code(), HTTP_101);
+                Assert.assertEquals(result.diagnosticInfo().code(), CompilerPluginTestConstants.HTTP_101);
```

Comment:
We can use static import and reduce the redundant use of `CompilerPluginTestConstants` through out this file